playbook/workshops/README.md

```diff
@@ -1,16 +1,80 @@
 # Workshops
 
-This directory will contain all of the workshops created for hackEDU.
+Below a list of all of our workshops.
 
-## Portfolio Workshop
+Our workshops aim to:
 
-**Notes:**
+- be targeted towards beginners and also accommodate for the more experienced
+- be focused on building something fun that you want share with your friends
+- get something shipped as quickly as possible
+- be modular enough to not feel like you have to go through them linearly
+- work well when done in pairs or individually
+- be tinkerable
 
-- This workshop has not been tested yet.
-- I anticipate it running 1hr - 2 hours long (it's too long).
+Each workshop is a markdown file and has associated facilitation guidelines.
 
-**Anticipated Issues:**
+See our general workshop facilitation guidelines
+[here](workshop_details.md#general-workshop-facilitation-guidelines):
 
-- People will be bored because of length
+Our workshops are free to be used by anyone. If you see any issues,
+[make an issue](https://github.com/hackedu/hackedu/issues/new) or even better  
+yet, make a [pull request](https://help.github.com/articles/using-pull-requests/)!
 
-[Link to workshop.](portfolio/README.md)
+---
```

Comment:
[According to the markdown laws, this should be 80 chars long.](https://github.com/hackedu/meta/blob/master/markdown_style_guide.md#horizontal-rules)